apps/react-18-tests-v9/package.json

```diff
@@ -20,6 +20,9 @@
   },
   "dependencies": {
     "@fluentui/react-components": "*",
+    "@testing-library/dom": "^10.1.0",
```

Comment:
cant we bump whole monorepo to these new versions ? ( in order to follow SVP(single version policy) )

Reply:
@Hotell nope...

- RTL@12 supports only React 17 & below
  <img width="375" alt="image" src="https://github.com/microsoft/fluentui/assets/14183168/ea287db6-7a8c-4450-b59f-aae0a47952ab">
- RTL@13 support only 18 & above
  <img width="352" alt="image" src="https://github.com/microsoft/fluentui/assets/14183168/79572fc0-b287-480d-a935-03ad51f4f6f0">

Reply:
okay :) 
![image](https://github.com/microsoft/fluentui/assets/1223799/67505d84-7fbd-4f92-b1e4-12b788f23fcd)